WcaOnRails/lib/combined_round_cutoff.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+class CombinedRoundCutoff
+  attr_accessor :numberOfAttempts, :cutoff
+  def initialize(numberOfAttempts: nil, cutoff: nil)
+    self.numberOfAttempts = numberOfAttempts
+    self.cutoff = cutoff
+  end
+
+  def cutoff=(new_cutoff)
+    @cutoff = Cutoff.load(new_cutoff)
+  end
+
+  def to_json_obj
+    { "numberOfAttempts" => self.numberOfAttempts, "cutoff" => self.cutoff&.to_json_obj }
+  end
+
+  def ==(o)
+    o.class == self.class && o.to_json_obj == self.to_json_obj
+  end
+
+  def hash
```

Comment:
It's not clear what these "bread & butter" functions are for. They don't seem to be used or tested.